docs/index.md

```diff
@@ -3,12 +3,146 @@
+You can also drag-and-drop files onto the upload location.
+
+<img src="img/configure_files_upload.png" width="450" alt="Click upload files icon" aria-label="Upload files link" title="Click or drag-and-drop to upload files"/>
+
+Next, select "Server Side" from the drop down on the upper right of the page, and click "New".
+
+<img src="img/configure_files_server_side.png" width="450" alt="Choose dropdown server-side then new" aria-label="Create new server-side definition" title="Create new server-side file configuration"/>
+
+This will display a new window showing the uploaded files.
+Change the name of the configuration if you want to.
+Select "OK" to complete the configuration and dismiss the window.
+
+<img src="img/configure_files_new_server_side.png" width="450" alt="Finish server-side configuration" aria-label="Click OK to complete configuration" title="Click OK to accept"/>
+
+The configured access is now available to workflows and shown on the page.
+
+<img src="img/configure_files_completed.png" width="450" alt="Configured file storage listing" aria-label="Configured file storage listing" title="Configured file storage listing"/>
+
+You can return to the main page by clicking the "back" button.
+
+<img src="img/configure_files_back.png" width="450" alt="Click back button" aria-label="Click back button for main page" title="Click the back button"/>
+
+### Running a workflow
+
+On the main page of the site, click the `Image` link under **Workflows**.
+
+<img src="img/landing_page_image.png" width="450" alt="Image workflow definition link" aria-label="Image workflow definition link" title="Click on Image link"/>
+
+Choose "Canopy Cover" from the drop down on the upper right of the page, and click "Run".
+
+<img src="img/workflows_image_cc_run.png" width="450" alt="Choose dropdown Canopy Cover then run" aria-label="Run Canopy Cover workflow" title="Run Canopy Cover workflow"/>
+
+The fields needed for running this workflow are shown on the page.
+We will be configuring the "Image file" field next.
+
+<img src="img/workflows_image_cc_fields.png" width="450" alt="Canopy Cover workflow fields" aria-label="Available Canopy Cover workflow fields" title="Canopy Cover workflow fields"/>
+
+Select the ellipses next to "Image file" to bring up the browsing window.
+Since we've  only defined one file configuration, it will be automatically chosen for us.
+Select the orthomosaic file and click "OK" to confirm your selection and dismiss the window.
+If you are using the sample data, the name of the file to choose is "sample_orthophoto.tif".
+
+<img src="img/workflows_image_cc_field_file.png" width="450" alt="Selecting image to process" aria-label="File selection window" title="Select image to process"/>
+
+Click the ellipses next to each of the other entries to choose the GeoJSON and YAML experiment files for this workflow.
```

Comment:
can this also take a shapefile? does that require multiple shape files be zipped?

Reply:
We made a decision a while ago to only accept GeoJSON files. I'm not sure, are you suggesting that we provide shapefile->GeoJSON functionality on the UI?

Reply:
>  decision a while ago to only accept GeoJSON files

Sorry if this didn't come across correctly - at the level of an algorithm, yes, it should only accept 'standard' data formats like geojson. _but_ having another tool that converts shp --> geojson to 'help users out' would also be helpful. At the UI level it might make sense to have this happen behind the scenes automatically, although this is also a chance to let them know 'that is not acceptable - let us convert it to an acceptable format for you'

Reply:
I will add an issue to add this functionality: https://github.com/AgPipeline/issues-and-projects/issues/556